Index: activemq-cpp/decaf/util/zip/Adler32.cpp
===================================================================
--- activemq-cpp/decaf/util/zip/Adler32.cpp	(revision 1355046)
+++ activemq-cpp/decaf/util/zip/Adler32.cpp	(working copy)
@@ -17,7 +17,7 @@
 
 #include "Adler32.h"
 
-#include <decaf/internal/util/zip/zlib.h>
+#include <zlib.h>
 
 using namespace decaf;
 using namespace decaf::lang;
Index: activemq-cpp/decaf/util/zip/Inflater.cpp
===================================================================
--- activemq-cpp/decaf/util/zip/Inflater.cpp	(revision 1355046)
+++ activemq-cpp/decaf/util/zip/Inflater.cpp	(working copy)
@@ -20,7 +20,7 @@
 #include <stdio.h>
 #include <assert.h>
 
-#include <decaf/internal/util/zip/zlib.h>
+#include <zlib.h>
 
 #include <decaf/lang/exceptions/RuntimeException.h>
 
Index: activemq-cpp/decaf/util/zip/CRC32.cpp
===================================================================
--- activemq-cpp/decaf/util/zip/CRC32.cpp	(revision 1355046)
+++ activemq-cpp/decaf/util/zip/CRC32.cpp	(working copy)
@@ -17,7 +17,7 @@
 
 #include "CRC32.h"
 
-#include <decaf/internal/util/zip/zlib.h>
+#include <zlib.h>
 
 using namespace decaf;
 using namespace decaf::lang;
Index: activemq-cpp/decaf/util/zip/Deflater.cpp
===================================================================
--- activemq-cpp/decaf/util/zip/Deflater.cpp	(revision 1355046)
+++ activemq-cpp/decaf/util/zip/Deflater.cpp	(working copy)
@@ -20,7 +20,7 @@
 #include <stdio.h>
 #include <assert.h>
 
-#include <decaf/internal/util/zip/zlib.h>
+#include <zlib.h>
 #include <decaf/lang/exceptions/RuntimeException.h>
 
 using namespace decaf;